install_dependencies.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 set -ex
-
-# TODO
+# export BUILD_ARCHITECTURE=arm64
+# Get ubuntu distribution code name. All STOL APT debian packages are pushed to S3 bucket based on distribution codename.
+echo "BUILD_ARCHITECTURE=${BUILD_ARCHITECTURE}"
+if [[ "${BUILD_ARCHITECTURE}" =~ arm* ]]; then
+    echo "WARNING: Python module support currently only exists for native x64 builds. Skipping Python module binding ... "
+else
+    /opt/carma/scripts/install_dependencies_script.sh python3-dev
```

Comment:
What happens if this depended install_dependecies_script.sh location changed? How is this scripted downloaded into this predefined path?

Reply:
Please look at the documentation in https://github.com/usdot-fhwa-stol/carma-builds . All debian package libraries use carma-builds base images to enable CPack functionality and cross-compiling for ARM debian packages